AdyenComponents/Apple Pay/ApplePayComponent.swift

```diff
@@ -37,6 +37,11 @@ public class ApplePayComponent: NSObject, PresentableComponent, PaymentComponent
     
     /// The delegate of the component.
     public weak var delegate: PaymentComponentDelegate?
+
+    public var payment: Payment? {
+        get { internalPayment }
+        set {} // swiftlint:disable:this unused_setter_value
```

Comment:
What's the reason we are not setting `internalPayment` here?

Reply:
maybe an assertionFailure?

Reply:
It will breach every integration since v4.0.0 😅
Will park for now..